docs/source/sysadmin_guide/configuring_permissions.rst

```diff
@@ -226,35 +286,39 @@ Network Roles
 The following is the suggested high-level role for on-chain validator network
 permissioning.
 
-network
+``default``:
+  When evaluating role permissions, if the role has not been set, the default
+  policy is used. The policy can be changed to meet the network's
+  requirements after initial start-up by submitting a new policy with the name
+  default. If the default policy has not been explicitly set, the default
+  is ``PERMIT_KEY \*``.
```

Comment:
Remove the "\"

Reply:
As above, I shall remove the now-unnecessary backslash.